spec/integration/request_controller_spec.rb

```diff
@@ -83,4 +83,31 @@
 
   end
 
+  describe 'FOI officer uploading a reponse' do
+
+    let(:user) { FactoryGirl.create(:user, :name => "Awkward > Name") }
+    let(:request) { FactoryGirl.create(:info_request, :user => user) }
+    let(:officer) { FactoryGirl.create(:user) }
+
+    before do
+      allow(request.public_body).
```

Comment:
I think this would be true anyway (aren't they both localhost?). Either way, since this is an integration test, I think it'd be better to set the request email and the user's email to the same domain, rather than mock things.